contracts/tokens/ERC1155/ERC1155.sol

```diff
@@ -3,14 +3,14 @@ pragma solidity ^0.6.8;
 import "../../utils/SafeMath.sol";
 import "../../interfaces/IERC1155TokenReceiver.sol";
 import "../../interfaces/IERC1155.sol";
-import "../../interfaces/IERC165.sol";
 import "../../utils/Address.sol";
+import "../../utils/ERC165.sol";
 
 
 /**
  * @dev Implementation of Multi-Token Standard contract
  */
-contract ERC1155 is IERC1155 {
+abstract contract ERC1155 is IERC1155, ERC165 {
```

Comment:
ERC1155 fully implements both interfaces, is there a reason why it's defined as `abstract`?

Reply:
oh, good catch, this is a mistake :). Forgot to undo that change when I was trying something.